agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -71,6 +71,8 @@ const (
 	dockerContainerDieEvent = "die"
 	// dockerContainerEventExitCodeAttribute is the attribute name to get exit code from Docker event attribute.
 	dockerContainerEventExitCodeAttribute = "exitCode"
+	// NetworkAliasDockerLabel is a nanme of docker label to set a network alias value
```

Comment:
Wording and spelling (typo) seems off here. Could we update this?

Current:
`is a nanme of docker label to set a network alias value`

Suggested:
`the name of the docker label to set network alias value`